README.md

```diff
@@ -1,8 +1,8 @@
 <p align="center">
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="docs/images/revup_dark.svg">
-  <source media="(prefers-color-scheme: light)" srcset="docs/images/revup_light.svg">
-  <img alt="Revup">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/Skydio/revup/main/docs/images/revup_dark.svg"/>
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/Skydio/revup/main/docs/images/revup_light.svg"/>
+  <img alt="Revup"/>
```

Comment:
Hmm, might want to use this media query for symforce, this is nice.  Have you confirmed this doesn't give you two copied of the image on pypi (probably via TestPyPI)?